compiler/src/iree/compiler/Dialect/Stream/Transforms/EncodeTensors.cpp

```diff
@@ -335,12 +336,38 @@ static Value canonicalizeFillPattern(Value pattern, PatternRewriter &rewriter) {
 
   // Get floats into integer form.
   auto patternType = pattern.getType();
-  unsigned bitWidth = patternType.getIntOrFloatBitWidth();
+  unsigned bitWidth = 0;
+  if (isa<ComplexType>(patternType)) {
+    bitWidth = dyn_cast<ComplexType>(patternType)
+                   .getElementType()
+                   .getIntOrFloatBitWidth();
+  } else {
+    bitWidth = patternType.getIntOrFloatBitWidth();
+  }
   if (patternType.isa<FloatType>()) {
     pattern = rewriter.createOrFold<arith::BitcastOp>(
         loc, rewriter.getIntegerType(bitWidth), pattern);
   }
 
+  if (isa<ComplexType>(patternType)) {
+    std::int64_t value = bitWidth;
```

Comment:
We should name this something other than value. E.g. `complexBitwidth`. We also can just directly use `int64_t`, not `std::` required.

Reply:
Fixed both.